libraries/Microsoft.Bot.Builder/Adapters/TestAdapter.cs

```diff
@@ -808,6 +836,11 @@ public Task<TokenResponse> ExchangeTokenAsync(ITurnContext turnContext, AppCrede
 
             if (_exchangableToken.TryGetValue(key, out string token))
             {
+                if (token == ExceptionExpected)
+                {
+                    throw new Exception("Exception occurred during exchanging tokens");
```

Comment:
Is there a better message we can put here? This doesn't exactly seem actionable.

Reply:
This is the test adapter...The unit test just expects an exception to be thrown